src/browser/js/WebappConnector.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2015-2016 Yuya Ochiai
```

Comment:
Should we omit Yuya from new files?

Reply:
Sounds like it's not necessary for new files ... I'll update. Thx.